vtex-apps/wordpress-integration
# wrike 460923202 - fixing issue with file name and interface

update file name from WordpressCategoryRelatedPosts to WordpressCategoryRelatedPostsBlock and change interface from blog-related-posts.wordpress-category-related-posts to blog-search-list.wordpress-category-related-posts so that block correctly works on search page

**What problem is this solving?**

reported "entry point not found" error for new block component.

**How should this be manually tested?**

working on this page (by displaying nothing). Component can be modified to display div with text in place of null on line 117 to verify block is returning elements correctly even without sample wordpress posts configured.

https://wordpressrelatedcattest--storecomponents.myvtex.com/shirt?_q=shirt&map=ft

**Screenshots or example usage:**
